.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Run Release Please
+
+on:
+  workflow_dispatch:
+    inputs:
+
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+
+    # Release-please creates a PR that tracks all changes
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          command: manifest
+          token: ${{secrets.GITHUB_TOKEN}}
+          default-branch: main
+          signoff: "OpenFeature Bot <109696520+openfeaturebot@users.noreply.github.com>"
+      - name: Dump Release Please Output
+        env:
+          RELEASE_PLEASE_OUTPUT: ${{ toJson(steps.release.outputs) }}
+        run: |
+          echo "$RELEASE_PLEASE_OUTPUT"
```

Comment:
Should we keep this dump step or is it only for debugging the first releases?

Reply:
I find this very helpful to debug issues when something goes off at release time 😅 
we can remove it if you prefer tho

Reply:
No, it's fine! Just wanted to have a record of why it's there.